scripts/generate-bindings.ps1

```diff
@@ -0,0 +1,163 @@
+# This script generates Rust bindings for the SymCrypt library using bindgen.
+# It can be called directly from the command line or via generate-all-bindings.ps1.
+#
+# Prerequisites:
+# - LLVM
+# - bindgen
+
+[CmdletBinding()]
+param(
+    [Parameter(Mandatory, HelpMessage="Current triple can be found by running 'clang -print-target-triple'")]
+    [string]$triple,
+    [Parameter(Mandatory, HelpMessage="Output directory for the generated bindings")]
+    [string]$outDir
+)
+
+$ErrorActionPreference = "Stop"
+$PSNativeCommandUseErrorActionPreference = $True
+
+# Init variables
+$SymCryptSysCrate = "$PSScriptRoot/../symcrypt-sys"
+$SymCryptSysCrate = $SymCryptSysCrate.Replace("\", "/")
+$wrapperHeader = "$SymCryptSysCrate/inc/wrapper.h"
+$targetName = $triple.Replace("-", "_")
+$bindingsFile = "$outDir/$targetName.rs"
+
+$supportedTargets = @(
+    "x86_64-pc-windows-msvc",
+    "aarch64-pc-windows-msvc",
+    "x86_64-unknown-linux-gnu",
+    "aarch64-unknown-linux-gnu"
+)
+
+if ($supportedTargets -notcontains $triple) {
+    Write-Error "Unsupported target: $triple. Supported targets: $supportedTargets"
+    exit 1
+}
+
+$importRules = @(
```

Comment:
>$importRules [](http://example.com/codeflow?start=0&length=12)

Is it possible to separate the allow list to a separate file? I think it would be easier to maintain rather than directly modifying the .ps1 script. Wondering your thoughts